subprojects/snapshots/src/main/java/org/gradle/internal/vfs/impl/FileHierarchySet.java

```diff
@@ -62,4 +43,7 @@ public FileHierarchySet invalidate(String path) {
 
     @CheckReturnValue
     FileHierarchySet invalidate(String path);
+
+    @CheckReturnValue
+    FileHierarchySet empty();
```

Comment:
Does this need to be public?

Reply:
It is currently used here: https://github.com/gradle/gradle/blob/7d4f7be09923d3339af87d1ef045b983037e958e/subprojects/snapshots/src/main/java/org/gradle/internal/vfs/impl/DefaultVirtualFileSystem.java#L173